frontend/src/pages/marketplace/dataset-marketplace-card/dataset-marketplace-card.component.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+    BarChartOutlined,
+    DatabaseOutlined,
+    EyeOutlined,
+    NumberOutlined,
+    ShareAltOutlined,
+    TeamOutlined,
+} from '@ant-design/icons';
+import { Button, Card, Descriptions, type DescriptionsProps, List, Space, Tag, Tooltip, Typography } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { Link, useNavigate } from 'react-router';
+import { LoadingSpinner } from '@/components/loading/loading-spinner/loading-spinner';
+import { useGetDataProductByIdQuery } from '@/store/features/data-products/data-products-api-slice';
+import { useGetDatasetByIdQuery } from '@/store/features/datasets/datasets-api-slice.ts';
+import type { DatasetContract } from '@/types/dataset';
+import { createDataProductIdPath, createDatasetIdPath } from '@/types/navigation.ts';
+import styles from './dataset-marketplace-card.module.scss';
+
+type Props = {
+    datasetId: string;
+};
+
+export function DatasetMarketplaceCard({ datasetId }: Props) {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+    const { data: dataset, isLoading } = useGetDatasetByIdQuery(datasetId, { skip: !datasetId });
```

Comment:
I would rather pass the dataset from the marketplace, instead of doing a query here again?
Because this puts unneeded pressure in the API.
Similar with the next call to get the data product.

After discussion we decided to extend the search for datasets to give it what we need here, except we don't want to add the `data_product_links` in the search call.
We want to load these when the user hovers over the tooltip instead